### ISSUE TITLE
Report error for native integer token as using alias target

### ISSUE THREAD

**Issue by cston** [dotnet/roslyn#42975]
Consider reporting an error for a native integer token as a using alias target:
```C#
using A = nint; // error?

class Program
{
    A F() => default;
}
```

**Comment by jaredpar** [comment]
Agree we should be reporting an error here. The equivalent scenario for `dynamic` will report an error:

> error CS0246: The type or namespace name 'dynamic' could not be found (are you missing a using directive or an assembly reference?)

**Comment by AlekseyTs** [comment]
Done with review pass (iteration 1)
